pyasic/miners/backends/antminer.py

```diff
@@ -351,6 +352,35 @@ async def _get_uptime(self, rpc_stats: dict = None) -> Optional[int]:
             except LookupError:
                 pass
 
+    async def _get_pools(self, rpc_pools: dict = None) -> List[PoolMetrics]:
+        if rpc_pools is None:
+            try:
+                rpc_pools = await self.rpc.pools()
+            except APIError:
+                pass
+
+        pools_data = []
+        if rpc_pools is not None:
+            try:
+                pools = rpc_pools.get("POOLS", [])
+                for index, pool_info in enumerate(pools):
+                    pool_data = PoolMetrics(
+                        accepted=pool_info.get("Accepted"),
+                        rejected=pool_info.get("Rejected"),
+                        get_failures=pool_info.get("Get Failures"),
+                        remote_failures=pool_info.get("Remote Failures"),
+                        active=pool_info.get("Stratum Active"),
+                        alive=pool_info.get("Alive"),
+                        url=pool_info.get("URL"),
+                        user=pool_info.get("User"),
+                        index=pool_info.get("Index")
```

Comment:
Use the "POOL" key, not the "Index".  Not sure why they keyed it this way, but that's the right value.
```
index=pool_info.get("POOL"),
```

Reply:
okay, Noted. ✅